canonical-schema.json

```diff
@@ -102,7 +102,7 @@
           , "required"   : ["description", "cases"]
           , "properties" :
                 { "description": { "$ref": "#/definitions/description" }
-                , "optional"   : { "$ref": "#/definitions/optional"    }
+                , "scenarios"  : { "$ref": "#/definitions/scenarios"   }
```

Comment:
For now, I've allowed the possiblity of also having scenarios at the test group level.